### ISSUE TITLE
Can't install gsl package when it is located in a non-standard path

### ISSUE THREAD

**Issue by mikecarniello** [RobinHankin/gsl#27]
(this issue is similar to issue #23 in some ways.)

* GSL is not installed in our RHEL server
* I downloaded and installed it my own directory path
* I confirmed it works, by using it in a small C program
* I hoped I set environment variables such that they would point to the paths where the GSL content is
* the install.packages("gsl") fails with:
        checking for gsl-config... no
        configure: error: gsl-config not found, is GSL installed?
        ERROR: configuration failed for package ‘gsl’

Any ideas? I'm thinking that if I can get the "install,packges" function to find my own GSL path, the rest of the installation could complete.

Evidence on these points is in the attached PDF file.

[gsl installation notes.pdf](https://github.com/RobinHankin/gsl/files/12468457/gsl.installation.notes.pdf)

**Comment by RobinHankin** [comment]
Thanks for this, and thanks for the PDF notes.  Installation is failing because it cannot find `gsl-config`.  What does `which gsl-config` report?  tbh I have never been clear on the precise environment that the configure scripts run in.  I guess the next step would be to try and install the package directly from the `.tar.gz` file, which might make it easier to diagnose the problem.

**Comment by mikecarniello** [comment]
Thanks for the useful responses! We can close this ticket, with some good news. Details are below.

The "tar.gz" approach didn't work, so I tried the "which" hint (not "witch hunt," a popular phrase in the US these days).

I modified the Linux $PATH variable to find "gsl-config." That enabled the install.packages command to work, and gsl was installed.

I tested the execution - using command-line R - by running the function from the vignette. That worked!

This work is documented in the attached PDF, perhaps useful for future users.

I do have one remaining issue, which is not in "gsl" scope and may not be resolvable.

The issue: All this worked in command-lin
[which gsl.pdf](https://github.com/RobinHankin/gsl/files/12490617/which.gsl.pdf)
e R, which I launch from a Linux shell. This is all very ENV variable dependent, but it does work. When I put the ENV varialbles into .bashrc, this breaks my launching of R Studio (I think my user library is messed up). When I don't put the variables into .bashrc, the gsl library use breaks because components not found.

I need to either find a solution to that - using R Studio with the "right" ENV variables, or just work command-line R.

Anyway, again - thanks for the help!

(I am not quite sure how to close, but I see a button "Close with comment" and I hope that's accurate!) 


[which gsl.pdf](https://github.com/RobinHankin/gsl/files/12490618/which.gsl.pdf)
